### DISCUSSION TITLE
SQLite Studio + DB Browser for SQLite

### DISCUSSION THREAD

**Discussion by AGoh2019** [pawelsalawa/sqlitestudio#3785] [Ideas]
### Details
Combine the SQLite Studio project and the DB Browser for SQLite project into a single combined tool.

So that users of SQLite Studio can visualize how the normalized tables are associated in the database.

**Comment by pawelsalawa** [comment]
Are you aware, that this is like saying "let's combine Firefox and Chrome"?

What I believe you're asking about is ERD (Entities Relationship Diagram)  feature and it is on the list to do: #2865

**Comment by AGoh2019** [comment]
Apologies for not being clear. Yes ERD,  I hope its coming up.
Andrew Goh S M

 

    On Tuesday, 2 June 2020, 07:28:47 pm GMT+8, Paweł Salawa <notifications@github.com> wrote:  
 
 


Are you aware, that this is like saying "let's combine Firefox and Chrome"?

What I believe you're asking about is ERD (Entities Relationship Diagram) feature and it is on the list to do: #2865

**Comment by John-foobar-Doe** [comment]
Any update on this? A diagram visualizing the relations between tables is really helpful...
